### PR TITLE
add list group

### PR DESCRIPTION
**Pull request by fadhilahfarhan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -120,6 +120,15 @@ <h1 class="display-4 fw-bold lh-1 mb-3">
...
       </div>
     </div>
+
+    <!-- LIST GROUP SECTION -->
+    <ul class="list-group">
+      <li class="list-group-item">An item</li>
+      <li class="list-group-item">A second item</li>
+      <li class="list-group-item">A third item</li>
+      <li class="list-group-item">A fourth item</li>
+      <li class="list-group-item">And a fifth one</li>
```
Comment (alamsyhhh) [inline index.html]: change the content "And a fifth one" into "A fifth item"

**Review by alamsyhhh — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -120,6 +120,15 @@ <h1 class="display-4 fw-bold lh-1 mb-3">
         </div>
       </div>
     </div>
+
+    <!-- LIST GROUP SECTION -->
+    <ul class="list-group">
+      <li class="list-group-item">An item</li>
```
Comment (mnizarzr) [inline index.html]: item pertama warna hijau
text-success

**Review by mnizarzr — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (fadhilahfarhan) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (fadhilahfarhan) [reply index.html]: done

**Review by alamsyhhh — APPROVED** [review]

**Review by mnizarzr — APPROVED** [review]

**Pull request merged**